python/pyspark/sql/readwriter.py

```diff
@@ -77,7 +77,7 @@ def _set_json_opts(self, schema, primitivesAsString, prefersDecimal,
     def _set_csv_opts(self, schema, sep, encoding, quote, escape,
```

Comment:
That's a good idea. There are a bunch of things I want to do to the readwrite.py (mainly break it apart). I will do it there and merge this to unblock the rc.